gnomad/utils/filtering.py

```diff
@@ -599,13 +613,26 @@ def filter_arrays_by_meta(
         )
 
     if isinstance(items_to_filter, list):
-        filter_func = lambda m, k: m.contains(k)
         items_to_filter = [[k] for k in items_to_filter]
+        items_to_filter_set = hl.set(items_to_filter)
```

Comment:
Sorry with the line 616 above, I dont think this works. I didnt realize that the list of lists was needed later on L643 the `meta_expr`.  Those if you just move line 617 above L616, it should?